lib/studies/reply/adaptive_nav.dart

```diff
@@ -581,6 +592,25 @@ class _MobileNavState extends State<_MobileNav> with TickerProviderStateMixin {
     }
   }
 
+  bool _handleScrollNotification(ScrollNotification notification) {
+    if (notification.depth == 0) {
+      if (notification is UserScrollNotification) {
+        final userScroll = notification;
```

Comment:
is this necessary to create a new variable here?

Reply:
It's only used once so I think we can do without it.